src/components/input/input.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/617c60ee1d.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../../artsyUI.css" />
+    <title>Artsy UI</title>
+</head>
+
+<body>
+    <!-- header -->
+    <header>
+        <div>
+            <img alt="artsyui-logo" class="artsy-logo" src="../../assets/logo-v2.png" />
+        </div>
+        <div class="right-side">
+            <a class="link-text-primary">Home</a>
+            <a class="link-text-primary">Docs</a>
+            <div class="mode-toggle">
+                <svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 512 512" aria-hidden="true"
+                    focusable="false" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M256 160c-52.9 0-96 43.1-96 96s43.1 96 96 96 96-43.1 96-96-43.1-96-96-96zm246.4 80.5l-94.7-47.3 33.5-100.4c4.5-13.6-8.4-26.5-21.9-21.9l-100.4 33.5-47.4-94.8c-6.4-12.8-24.6-12.8-31 0l-47.3 94.7L92.7 70.8c-13.6-4.5-26.5 8.4-21.9 21.9l33.5 100.4-94.7 47.4c-12.8 6.4-12.8 24.6 0 31l94.7 47.3-33.5 100.5c-4.5 13.6 8.4 26.5 21.9 21.9l100.4-33.5 47.3 94.7c6.4 12.8 24.6 12.8 31 0l47.3-94.7 100.4 33.5c13.6 4.5 26.5-8.4 21.9-21.9l-33.5-100.4 94.7-47.3c13-6.5 13-24.7.2-31.1zm-155.9 106c-49.9 49.9-131.1 49.9-181 0-49.9-49.9-49.9-131.1 0-181 49.9-49.9 131.1-49.9 181 0 49.9 49.9 49.9 131.1 0 181z">
+                    </path>
+                </svg>
+            </div>
+        </div>
+    </header>
+
+    <!-- aside start -->
+    <aside>
+        <!-- Getting Started -->
+        <h3>Getting Started</h3>
+        <a href="">Introduction</a>
+        <a href="">Typography</a>
+
+        <!-- components -->
+        <h3>Components</h4>
+            <a href="../alert/alert.html">Alert</a>
+            <a href="../avatar/avatar.html">Avatar</a>
+            <a href="../badge/badge.html">Badge</a>
+            <a href="../button/button.html">Button</a>
+            <a  href="../card/card.html">Card</a>
+            <a href="../image/image.html">Image</a>
+            <a class="active-tab" href="">Input</a>
+            <a href="">Form</a>
+            <a href="">Snackbar</a>
+    </aside>
+
+    <!-- main -->
+    <main>
+        <h1>Input</h1>
+        <p class="body-l"></p>
+
+        <!-- inputs -->
+        <div class="component-desc">
+            <h3>Text Input</h3>
+            <p>By default artsy ui inputs are outline styled, to change to standard style input add class <span class="highlight-text">input-std</span> to the input.<br>
+            To mark a input as req just add class <span class="highlight-text">input-req</span> to the div. </p>
```

Comment:
try to not use abbreviations in descriptions of components. here, "req" could be changed to "required"